ui/app/components/app/dai-migration-component/dai-migration-notification.component.js

```diff
@@ -8,18 +9,46 @@ export default class DaiV1MigrationNotification extends PureComponent {
   }
 
   static defaultProps = {
+    mkrMigrationReminderTimestamp: null,
     string: '',
     symbol: '',
   }
 
   static propTypes = {
+    setMkrMigrationReminderTimestamp: PropTypes.func.isRequired,
+    mkrMigrationReminderTimestamp: PropTypes.string,
     string: PropTypes.string,
     symbol: PropTypes.string,
   }
 
+  state = {
+    dismissed: false,
+  }
+
+  remindMeLater = () => {
+    const nextWeek = DateTime.utc().plus({
+      seconds: 60,
```

Comment:
I'm guessing this is from testing? :sweat_smile:

Reply:
Yes 😅